tests/rt.conf

```diff
@@ -142,8 +142,9 @@ RUN     | control_wam_debug
 # HAFS tests                                                                                                                                                                      #
 ###################################################################################################################################################################################
 #
-COMPILE | -DAPP=HAFSW -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_gfdlmp_tedmf_nonsst -D32BIT=ON                           | - wcoss2                                | fv3 |
+COMPILE | -DAPP=HAFSW -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_thompson_tedmf_gfdlsf,FV3_HAFS_v0_thompson_tedmf_gfdlsf_nonsst,FV3_HAFS_v0_gfdlmp_tedmf_nonsst -D32BIT=ON                           | - wcoss2                                | fv3 |
```

Comment:
@ChunxiZhang-NOAA, could you change this line into the following? Thanks!
COMPILE | -DAPP=HAFSW -DCCPP_SUITES=FV3_HAFS_v0_gfdlmp_tedmf,FV3_HAFS_v0_gfdlmp_tedmf_nonsstFV3_HAFS_v0_thompson_tedmf_gfdlsf,FV3_HAFS_v0_thompson_tedmf_gfdlsf_nonsst, -D32BIT=ON                           | - wcoss2                                | fv3 |

Reply:
Changed as you suggested.